content/docs/ops/create-org.md

```diff
@@ -14,11 +14,16 @@ New sandbox organizations are automatically created when users sign in for the f
 Non-sandbox organizations need to be created by hand. To do so:
 
 1. The [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1) should be filled out by you or the tenant.
-1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script.
+1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script. You will need the following information, some of which is in the [form output](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=131031416)to pass to the script as arguments:
+   * **Agency**: the requesting agency, column B of the form output. 
+   * **Unique business ID**: currently provided by first-tier support.
+   * **System name**: the agency's system, column C of the form output.
+   * **Note**: currently provided by first-tier support.
+   * **Memory**: 4G by default; this should be set to a reasonable initial cap for the systems expected to live in that quota so that clients aren't shocked at their first bill. They can ask for increases as needed.
+NOTE: We don't yet have a standard convention for what constitutes a unique business ID nor what to put in the note field.
```

Comment:
> we can use the org+system (columns E+D) to find the right line of column H and use that for the business ID

👍  The thinking with linking to that doc in the form is that the onus is on the tenant (at least for teams in 18F who can access the Agreements list) to identify the right row.

> the jury's still out on the "note" field contents

Agreed! I was just following https://github.com/18F/cg-site/issues/219#issuecomment-243959351.